### PR TITLE
MFW discord.js somehow broke itself without ever updating

### PR DESCRIPTION
**Pull request by IllusionMan1212**

### CODE REVIEW

File: src/Command.ts [hunk 1]
```diff
@@ -96,7 +96,7 @@ export abstract class Command
                         opt.setName(arg.name)
                             .setDescription(arg.desc)
                             .setRequired(arg.required ?? false)
-                            .addChannelTypes(ChannelType.GuildText, ChannelType.GuildNews, ChannelType.GuildPublicThread, ChannelType.GuildPrivateThread, ChannelType.GuildVoice)
+                            .addChannelTypes(ChannelType.GuildText, ChannelType.GuildAnnouncement, ChannelType.PublicThread, ChannelType.PrivateThread, ChannelType.GuildVoice)
```
Comment (canarado) [inline src/Command.ts:99]: This is fucking terrible.

File: src/settings.ts [hunk 2]
```diff
@@ -79,7 +79,7 @@ export function getEnvVar(varName: string, asType: "stringArray"): undefined | s
 /** Grabs an environment variable's value, and casts it to a `number[]` */
 export function getEnvVar(varName: string, asType: "numberArray"): undefined | number[]
 /** Grabs an environment variable's value, and casts it to a specific type (default string) */
-export function getEnvVar<T extends ("string" | "number" | "stringArray" | "numberArray")>(varName: string, asType: T = "string" as T): undefined | (string | number | string[] | number[])
+export function getEnvVar<T extends ("string" | "number" | "stringArray" | "numberArray" | "stringNoEmpty")>(varName: string, asType: T = "string" as T): undefined | (string | number | string[] | number[])
```
Comment (canarado) [inline src/settings.ts:82]: WUT

**Review by canarado — APPROVED** [review]

File: src/settings.ts (hunk 2 above)
Comment (IllusionMan1212) [reply src/settings.ts:82]: I don't know mane. It was bitching about that not being an option so I added it. It's Sv magic code so I don't touch it

File: src/Command.ts (hunk 1 above)
Comment (IllusionMan1212) [reply src/Command.ts:99]: 🤷

File: src/settings.ts (hunk 2 above)
Comment (IllusionMan1212) [reply src/settings.ts:82]: it no longer bitches but im still gonna keep the change cuz its broken in TS 5.2

Comment (Sv443) [reply src/settings.ts:82]: yea I actually knew that was a bug, just not that it would actually error

**Review by Sv443 — APPROVED** [review]
chungos

**Pull request merged**